front/src/applications/editor/tools/switchEdition/types.ts

```diff
@@ -77,15 +80,46 @@ const trackNodeTypeOrder = [
 ];
 
 export const useSwitchTypes = (infraID: number | undefined) => {
```

Comment:
This hook doesn't belong to a types.ts file. Could you move it to a useSwitchTypes.ts file ?